composer.json

```diff
@@ -15,13 +15,14 @@
     }
   ],
   "require": {
-    "php": ">=5.2",
+    "php": ">=5.3.3",
     "ext-curl": "*",
     "ext-json": "*",
     "ext-mbstring": "*"
   },
   "require-dev": {
-    "simpletest/simpletest": "*"
+    "phpunit/phpunit": "~4.0",
+    "squizlabs/php_codesniffer": "~1.5"
```

Comment:
We should not be running cs checks on travis. Use scrutinizer or something to do that.